nautobot_chatops/api/urls.py

```diff
@@ -23,11 +23,11 @@
         path("ms_teams/messages/", MSTeamsMessagesView.as_view(), name="ms_teams_messages"),
     ]
 
-if settings.PLUGINS_CONFIG["nautobot_chatops"].get("enable_webex_teams"):
-    from nautobot_chatops.api.views.webex_teams import WebExTeamsView
+if settings.PLUGINS_CONFIG["nautobot_chatops"].get("enable_webex"):
```

Comment:
Can we make this backwards-compatible for a release or two? Something like:

```suggestion
if (
    settings.PLUGINS_CONFIG["nautobot_chatops"].get("enable_webex") or 
    settings.PLUGINS_CONFIG["nautobot_chatops"].get("enable_webex_teams")
):
    if (
        "enable_webex_teams" in settings.PLUGINS_CONFIG["nautobot_chatops"] and
        "enable_webex" not in settings.PLUGINS_CONFIG["nautobot_chatops"]
    ):
        logger.warning("The 'enable_webex_teams' setting is deprecated. Please use 'enable_webex' instead.")
```

(You'll of course need to add `import logging` and `logger = logging.getLogger(__name__)` at the head of this file.)

Reply:
Good idea.  Updated.